Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -183,6 +234,7 @@ def options_handler():
     parser.add_argument('--cloud_servers_path', help='Path to secret cloud server metadata file.')
     parser.add_argument('--cloud_servers_api_keys', help='Path to the file with cloud Servers api keys.')
     parser.add_argument('--unremovable_packs', help='List of packs that cant be removed.')
+    parser.add_argument('--one-by-one', help='', action='store_true')
```

Comment:
What does it mean ` action='store_true'`?
Please add a help description.

Reply:
@daryakoval store_true means this flag is boolean (https://docs.python.org/3/howto/argparse.html#introducing-optional-arguments)
Added the help to the argument.